### PR TITLE
LPPC FIR Boundary fix + GCCO Addition

### PR DESCRIPTION
**Pull request by dsolesvik**
As discussed in #413 and #414, this PR merges both.
The changes approved by Portugal vACC staff and VATSIM Spain staff (myself - am on the Spain staff list and have sent the email to Nestor to get approved, can re-send the email to the new email if necessary).
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+QGIS/URAL.shp
+QGIS/URAL.shx
+QGIS/lppcgcco.cpg
+QGIS/lppcgcco.dat
+QGIS/lppcgcco.dbf
+QGIS/LPPCGCCO.geojson
+QGIS/lppcgcco.prj
+QGIS/lppcgcco.qmd
+QGIS/lppcgcco.shp
+QGIS/lppcgcco.shx
```
Comment (Adrian2k) [inline .gitignore:38]: I assume this was not meant to be added?

**Review by Adrian2k — COMMENTED** [review]
We have you on the approved list now, but there are some changes done to the gitignore file.

File: .gitignore (hunk 1 above)
Comment (dsolesvik) [reply .gitignore:38]: Yes, sorry, forgot to uncheck this when pushing the PR. How do I go about removing this change from the PR, not too familiar with Git :P.

Comment (NelisV) [reply .gitignore:38]: image

**Comment by dsolesvik** [comment]
Updated!

**Review by Adrian2k — APPROVED** [review]

**Review by NelisV — APPROVED** [review]

**Pull request merged**